test/RibbonThetaVaultYearn.js

```diff
@@ -123,7 +159,7 @@ function behavesLikeRibbonOptionsVault(params) {
           {
             forking: {
               jsonRpcUrl: process.env.TEST_URI,
-              blockNumber: 12474917,
+              blockNumber: 12655142,
```

Comment:
skipped yvweth tests because changing block number caused some issues, but had to be done because usdc yearn pricer was deployed later

Reply:
One way around this is to specify a block number and reset the fork for every vault test block